react/react1/week2/todo-app/src/App.js

```diff
@@ -0,0 +1,37 @@
+import './App.css';
+import AddTodo from './components/AddTodo';
+import Header from './components/Header';
+import Timer from './components/Timer';
+import TodoList from './components/TodoList';
+import { todos } from './todos';
+import { useState } from 'react';
+import { v4 as id } from 'uuid';
+
+function App() {
+  const [todoList, setTodoList] = useState(todos);
+
+  function addTodo(description) {
+    if (description.trim()) {
+      setTodoList([...todoList, { id: id(), description }]);
```

Comment:
Wonderful solution, but please notice that you are trimming your description only on the condition check and not in the setting of new TODO